server/controllers/middlewares/auth/index.ts

```diff
@@ -1,19 +1,15 @@
 import { RequestHandler } from 'express';
 import dotenv from 'dotenv';
-import { verify } from 'jsonwebtoken';
+import { jwtVerify } from '../../../utils/jwt';
 
 dotenv.config();
 
-const checkAuth: RequestHandler = async (req, res, next) => {
+const checkAuth: RequestHandler = async (req: any, res, next) => {
   const { token } = req.cookies;
   if (!token) return res.status(401).json('UnAuthraised');
```

Comment:
Use the `CustomError` here since this is the pattern we stick to it in this project.